### PR TITLE
Add script to run validator

### PR DESCRIPTION
**Pull request by ales-tsurko**
As nodes are now running with safe only RPC API's, we need to run it once with unsafe RPC-API's enabled to add session keys. Then we restart it again with safe-only API's. The script is added to run a node with unsafe RPC-API's enabled, add keys and perform `staking->validate` call.

### CODE REVIEW

File: deploy/testnet/docker-compose.yml [hunk 1]
```diff
@@ -121,34 +121,6 @@ services:
     networks:
       - sportchain-node-network
 
-  certbot:
-    image: certbot/certbot
-    volumes:
-      - /etc/letsencrypt:/etc/letsencrypt
-      - ./certbot/www:/var/www/certbot
-    entrypoint: "/bin/sh -c 'trap exit TERM; certbot certonly --standalone --preferred-challenges http -d testnet-rpc.atleta.network --email admin@moneyfactory.io --agree-tos --non-interactive --verbose; sleep 12h'"
```
Comment (hrls) [inline deploy/testnet/docker-compose.yml:129]: Looks like env vars defined below (email and url) can be used here as well.
Style note: define `entrypoint` section as the last section and use multiline.

Comment (ales-tsurko) [reply deploy/testnet/docker-compose.yml:129]: it's removed

File: .github/workflows/deploy-testnet.yml [hunk 2]
```diff
@@ -6,124 +6,124 @@ on:
...
+      
+    - name: Set timestamp variables
+      run: |
+          DATE=$(date +%d%m%y)
+          HMS=$(date +%H%M%S)
+          echo "DATE: $DATE, HMS: $HMS"
+
+    - name: Build and push docker image
+      run: |
+          docker build -t docker.io/atletanetwork/atleta-node:testnet-${DATE}-${HMS} .
```
Comment (hrls) [inline .github/workflows/deploy-testnet.yml:34]: IMO something predefined (like commit hash) is more suitable here.

File: .github/workflows/deploy-testnet.yml [hunk 3]
```diff
@@ -6,124 +6,124 @@ on:
...
+        mkdir -p ~/.ssh/
+        touch ~/.ssh/known_hosts
+        ssh-keyscan ${{ secrets.BOOTNODE_HOST }} >> ~/.ssh/known_hosts
+        echo "${{ secrets.SSH_PRIVATE_KEY }}" > ~/.ssh/id_rsa
+        chmod 600 ~/.ssh/id_rsa
+        eval $(ssh-agent)
+
+    - name: Copy files to the server
+      run: |
+        scp deploy/testnet/run-bootnode.sh ${{ secrets.BOOTNODE_USER}}@${{ secrets.BOOTNODE_HOST }}:~/atleta/run.sh
```
Comment (hrls) [inline .github/workflows/deploy-testnet.yml]: Extra space before `}}`

**Review by hrls — DISMISSED** [review]

**Review by DmitriVT — DISMISSED** [review]

File: .github/workflows/deploy-testnet.yml (hunk 2 above)
Comment (ales-tsurko) [reply .github/workflows/deploy-testnet.yml:34]: It was discussed with the devops who initially configured it. We needed an incrementing version. So we stopped on timestamps.

File: .github/workflows/deploy-testnet.yml (hunk 3 above)
Comment (ales-tsurko) [reply .github/workflows/deploy-testnet.yml]: Thanks)

File: .github/workflows/deploy-testnet.yml (hunk 2 above)
Comment (hrls) [reply .github/workflows/deploy-testnet.yml:34]: Maybe we can add commit hash as image metadata or annotation?

**Review by DmitriVT — DISMISSED** [review]

File: .github/workflows/deploy-testnet.yml (hunk 2 above)
Comment (ales-tsurko) [reply .github/workflows/deploy-testnet.yml:34]: Why? If you think, we really need it, please open a ticket. As we need to merge it today and do more work to upgrade runtime today

**Pull request merged**